.gitignore

```diff
@@ -34,6 +34,13 @@
 # Python byte code
 *.pyc
 
+# From PrecomputedConstraintCorrection
```

Comment:
Could you make another PR for this file?

Reply:
Sure :)

Reply:
https://github.com/sofa-framework/sofa/pull/3344

Reply:
Thanks!